outfitter-mobile-app/app.json

```diff
@@ -0,0 +1,27 @@
+{
+  "expo": {
+    "name": "outfitter-mobile-app",
+    "description": "This project is really great.",
```

Comment:
Can we change the description here?

Reply:
Commit 9caac9d0de137eeddd510d363c661e73dbc931ed hasn't addressed this. Are we planning on keeping this description as-is?

Reply:
oh yeah forgot about this. What description do we want?

Reply:
Honestly, would it hurt just to delete it?